api/views/phones.py

```diff
@@ -444,17 +451,37 @@ def resend_welcome_sms(request):
     return resp
 
 
+def _try_delete_from_twilio(message):
+    try:
+        message.delete()
+    except TwilioRestException as e:
+        # Raise the exception unless it's a 404 indicating the message is already gone
+        if e.status != 404:
+            raise e
+
+
 @decorators.api_view(["POST"])
 @decorators.permission_classes([permissions.AllowAny])
 @decorators.renderer_classes([TemplateTwiMLRenderer])
 def inbound_sms(request):
     incr_if_enabled("phones_inbound_sms")
     _validate_twilio_request(request)
+
+    """
+    TODO: delete the message from Twilio; how to do this AFTER this request? queue?
+    E.g., with a django-celery task in phones.tasks:
```

Comment:
I'm more inclined to try [rq](https://python-rq.org/) than celery, but we can cross that bridge when we implement tasks.